freqtrade/strategy/informative_decorator.py

```diff
@@ -54,12 +56,11 @@ def populate_indicators_1h(self, dataframe: DataFrame, metadata: dict) -> DataFr
     _timeframe = timeframe
     _fmt = fmt
     _ffill = ffill
+    _candle_type = CandleType.from_string(candle_type) if candle_type else None
```

Comment:
be nice if we could also use an enum instead of having to use a string

Reply:
you can for sure use an enum (`CandleType(CandleType.FUTURES)` returns CandleType.FUTURES) - but it's user-provided, so we can't rely on it - therefore we assume it's a string and "convert from string".